roles/htcondor/tasks/install_python_pip.yaml

```diff
@@ -0,0 +1,14 @@
+---
+# file: install_python_pip.yaml.
+
+- name: get python pip installation module
```

Comment:
double check need for python2 pip

Reply:
Also, does HTCondor need anything python?

Reply:
Seems like the poller uses python3 packages so I should be good to remove the python2 installations